storage/backend/nfs/mount.go

```diff
@@ -0,0 +1,72 @@
+package nfs
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/contiv/errored"
+	"github.com/contiv/volplugin/storage"
+)
+
+const (
+	mountInfoFile     = "/proc/self/mountinfo"
+	nfsMajorMountType = "0"
+)
+
+// Mounted shows any volumes that belong to volplugin on the host, in
+// their native representation. They yield a *Mount.
+func (d *Driver) Mounted(timeout time.Duration) ([]*storage.Mount, error) {
+	mounts := []*storage.Mount{}
+
+	content, err := ioutil.ReadFile(mountInfoFile)
+	if err != nil {
+		return nil, err
+	}
+
+	lines := strings.Split(string(content), "\n")
+	for _, line := range lines {
+		if len(line) == 0 {
+			continue
+		}
+
+		parts := strings.Split(line, " ")
+		devParts := strings.Split(parts[2], ":")
+		if len(devParts) != 2 {
+			return nil, errored.Errorf("Could not parse %q properly.", mountInfoFile)
+		}
+
+		if devParts[0] == nfsMajorMountType && parts[8] == "nfs4" {
```

Comment:
How do you feel about storing these parts[]/devParts[] into named variables?  It'd make this code a *lot* more readable IMO.  For example, `majorId` communicates a lot more than `devParts[0]`, and you don't have to repeat the array indexes multiple times, which is getting close to magic number territory.

Having the variables all grouped together would allow someone, at a glance, to see what the important parts of the chopped up string actually are.